components/admin_console/team_channel_settings/channel/details/channel_details.tsx

```diff
@@ -95,18 +104,25 @@ export default class ChannelDetails extends React.Component<ChannelDetailsProps,
 
         // If we don't have the team and channel on mount, we need to request the team after we load the channel
         if (!prevProps.team.id && !prevProps.channel.team_id && channel.team_id) {
-            this.props.actions.getTeam(channel.team_id);
+            this.props.actions.getTeam(channel.team_id).
+                then((data: any) => this.props.actions.loadScheme(data.data.scheme_id));
         }
```

Comment:
I think this block is not needed anymore since you call `getTeam` in `componentDidMount`

Reply:
I tried removing this but the reason I kept it in is because if I'm on the `Channel Configuration` page and reload it, it seems to not have loaded the team in time so I was getting linked to the System Scheme instead of the Team scheme.